src/util.cc

```diff
@@ -481,6 +481,32 @@ string StripAnsiEscapeCodes(const string& in) {
 
 int GetProcessorCount() {
 #ifdef _WIN32
+#if _WIN32_WINNT >= 0x0601
```

Comment:
https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=vs-2019
```suggestion
#if _WIN32_WINNT >= _WIN32_WINNT_WIN7
```